app/services/closure_decision_tree.rb

```diff
@@ -10,7 +10,9 @@ def destination
     when :additional_info
       edit(:support_documents)
     when :support_documents
-      show(:start) # TODO: check your answers step
+      show(:check_answers)
+    when :check_answers
+      task_list
```

Comment:
Hrm, there shouldn't be much left in the appeal flow, I think @tatyree took all of them out?We'll check later.

This should probably go to `root_path` for now, and then we change it when we implement the confirmation page.